src/js/modal-download.js

```diff
@@ -97,7 +97,7 @@ function actionModalFromClick (evt) {
 		if (visible()) {
 			const action = evt.target.getAttribute(ATTR_ACTION);
 
-			if (evt.target.matches(`.n-syndication-modal-shadow`) || (action && action === 'close')) {
+			if (evt.target.matches('.n-syndication-modal-shadow') || (action && action === 'close')) {
```

Comment:
Thank you 🙏

Reply:
I don't think I have touched those? But you're welcome anyway... :)